### PR TITLE
add reserved id for platform with default configurations

### PR DESCRIPTION
**Pull request by shubhbapna**
**Thank you for submitting this pull request**

**JIRA**: _(please edit the JIRA link if it exists)_

[link](https://www.example.com)

**referenced Pull Requests**: _(please edit the URLs of referenced pullrequests if they exist)_

- paste the link(s) from GitHub here
- link 2
- link 3 etc.

### CODE REVIEW

**Review by Ginxo — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -409,7 +409,7 @@ You can define multiple git platforms and define which platform to use for each
 ```
 platforms:
   - name: [OPTIONAL] Human readable name of the platform (eg: GitHub enterprise)
-  - id: Unique identifier of the platform. Used to map a platform to a project. Used in dependencies section
+  - id: Unique identifier of the platform. Used to map a platform to a project. Used in dependencies section. The ids: default-github and default-gitlab are reserved and are configured to use the default values.
```
Comment (lampajr) [inline README.md:412]:
```suggestion
  - id: Unique identifier of the platform. Used to map a platform to a project. Used in dependencies section. The ids: github-public and gitlab-public are reserved and are configured to use the default values.
```

Comment (lampajr) [reply README.md:412]: If I correctly understood we should update these too ,right?

**Review by lampajr — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (lampajr) [reply README.md:412]: Apart from this minor, it looks great to me

Comment (shubhbapna) [reply README.md:412]: Oh yes missed this, thanks!

**Review by lampajr — APPROVED** [review]

**Pull request merged**